dtfs-central-api/src/helpers/amendments/completed-amendments.ts

```diff
@@ -0,0 +1,9 @@
+import { AMENDMENT_STATUS, TfmFacilityAmendment } from '@ukef/dtfs2-common';
+
+/**
+ * Filters amendments to only those which are completed
+ * @param tfmFacilityAmendments - amendments to filter
+ * @returns completed amendments
+ */
+export const getCompletedAmendments = (tfmFacilityAmendments: TfmFacilityAmendment[]): TfmFacilityAmendment[] =>
```

Comment:
How about `filterCompletedAmendments?

Can we have the file name aligned with the function name as well please